boards/arm/mimxrt685_evk/doc/index.rst

```diff
@@ -227,6 +227,20 @@ configured by default to use the LPC-Link2.
 
 .. tabs::
 
+    .. group-tab:: LinkServer CMSIS-DAP
+
+        1. Install the :ref:`linkserver-debug-host-tools` and make sure they are in your
+           search path.  LinkServer works with the default CMSIS-DAP firmware included in
+           the on-board debugger.
+        2. Make sure the jumpers JP17, JP18 and JP19 are installed.
+
+        linkserver is the default runner for this board
```

Comment:
When I test this PR locally, linkserver does not appear to be the default runner:
```
(.venv) ~/zephyr/zephyrproject/zephyr:(b7a6ec1151...) ❯ west flash -r linkserver
-- west flash: rebuilding
[1/1] cd /home/danieldegrasse/zephyr/zephyrproject...ephyr/zephyrproject/zephyr/build/zephyr/zephyr.elf
-- west flash: using runner linkserver
-- runners.linkserver: LinkServer: /usr/local/LinkServer/LinkServer, version v1.3.15
INFO: Exact match for MIMXRT685S:EVK-MIMXRT685 found
INFO: Selected device MIMXRT685S:EVK-MIMXRT685
INFO: Getting available probes
INFO: Selecting probe by index
INFO: Selected probe #1 HQAQCQHR (LPC-LINK2 CMSIS-DAP V5.361)
(.venv) ~/zephyr/zephyrproject/zephyr:(b7a6ec1151...) ❯ west flash
-- west flash: rebuilding
[1/1] cd /home/danieldegrasse/zephyr/zephyrproject...ephyr/zephyrproject/zephyr/build/zephyr/zephyr.elf
-- west flash: using runner jlink
-- runners.jlink: reset after flashing requested
-- runners.jlink: JLink version: 7.92
-- runners.jlink: Flashing file: /home/danieldegrasse/zephyr/zephyrproject/zephyr/build/zephyr/zephyr.bin
FATAL ERROR: command exited with status 1: /opt/SEGGER/JLink_V792/JLinkExe -nogui 1 -if swd -speed auto -device MIMXRT685S_M33 -CommanderScript /tmp/tmpxtx7m3hxjlink/runner.jlink -nogui 1
(.venv) ~/zephyr/zephyrproject/zephyr:(b7a6ec1151...) ❯
```

I believe we need to update the [linkserver.board.cmake](https://github.com/zephyrproject-rtos/zephyr/blob/main/boards/common/linkserver.board.cmake#L1) file to include board_set_flasher_ifnset directives, like other runner cmake files do: [main/boards/common/jlink.board.cmake#L3](https://github.com/zephyrproject-rtos/zephyr/blob/main/boards/common/jlink.board.cmake?rgh-link-date=2023-12-21T20%3A30%3A05Z#L3). Otherwise, linkserver will not actually become the default